cmd/codegen/generator/open_api.go

```diff
@@ -166,7 +166,7 @@ type schemaWriterTemplateData struct {
 // InstallOpenApiGen installs the openapi-gen tool from the github.com/kubernetes/kube-openapi repository.
 func InstallOpenApiGen(version string, gopath string) error {
 	util.AppLogger().Infof("installing %s with version %s via 'go get' to %s", openApiGen, version, gopath)
-	err := util.GoGet(openApiGen, version, gopath, false, false, true)
+	err := util.GoGet(openApiGen, version, gopath, true, false, true)
```

Comment:
@pmuir - you remember https://github.com/jenkins-x/jx/pull/4359/files? This change from @abayer is going to rewind this. Can you remember the details? I am trying to remember why we disabled modules again and what side effects we might get with this PR.